utils/wui.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+import gzip
+import shutil
+
+class WUIFiles:
+    def __init__(self):
+        self.wui_files = []
+        # path to resource files directory
+        self.wui_path = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / 'src_local'
+        # path to main C file with raw data
+        self.main_file_name = 'fsdata_wui_local.c'
+        self.raw_data_file = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / self.main_file_name
+        self.raw_data_file_tmp = Path(__file__).resolve().parent.parent / 'lib' / 'WUI' / 'resources' / str(self.main_file_name + '_tmp')
+
+        # start
+        self.getFiles()
+        self.gzipFiles()
+        self.generateRawDataFile()
+    #endef
+
+    # store paths to static wui files in array for later use
```

Comment:
/suggestion/ Use python docstring for documentation

```Python
def getFiles(self):
    """ Store paths to static wui files in array for later use """
    ...
```

Reply:
I see you changed the way methods are documented. However, the docstrings have to be _under_ the function name (as in the example I gave you above). See https://www.python.org/dev/peps/pep-0257/.

Reply:
That blame should be on yapf side, because previous build has failed on pre-commit check and yapf did that.